tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -13,6 +13,8 @@
 using Windows.Win32;
 using Windows.Win32.UI.Input.KeyboardAndMouse;
 using WinUIEx;
+
+using static DevHome.PI.Helpers.CommonHelper;

```

Comment:
Doesn't look like this was needed? (No other changes in the file)

Reply:
Deleted this code and I haven't seen any negative repercussions so far.